app/controllers/profiles_controller.rb

```diff
@@ -10,19 +10,36 @@ def search
   def search_results
     @profile_results = search_profiles(params[:search])
   end
-
+  
+  private
+  
   def search_profiles(params)
     all = Profile.unscoped
     
-    if params["ethnicity"]
+    if params["ethnicity"] && params["ethnicity"] != ""
```

Comment:
Found that this didn't work, I think because in the second half we're checking that it's **not** blank?